### ISSUE TITLE
Problem dependencies with strapi@4.15.0

### ISSUE THREAD

**Issue by jpouvrard** [mattmilburn/strapi-plugin-menus#136]
I'm using `"@strapi/strapi": "4.15.0"`.
When I try to install `"strapi-plugin-menus": "1.5.0"`, I have this error:

```
Could not resolve dependency:
 strapi-plugin-menus@"1.5.0" from the root project
 Conflicting peer dependency: react-intl@6.5.1
 node_modules/react-intl
   peer react-intl@"^6.4.4" from strapi-plugin-menus@1.5.0
   node_modules/strapi-plugin-menus
     strapi-plugin-menus@"1.5.0" from the root project
```

Because of that:

```
Found: react-intl@6.4.1
 node_modules/react-intl
   react-intl@"6.4.1" from @strapi/admin@4.15.0
   node_modules/@strapi/admin
     @strapi/admin@"4.15.0" from @strapi/strapi@4.15.0
     node_modules/@strapi/strapi
       @strapi/strapi@"4.15.0" from the root project
```

**Comment by bora89** [comment]
Yeah, same for me

**Comment by mattmilburn** [comment]
Hi @jpouvrard @bora89 This will be fixed in an upcoming release 👍🏻  I'll ping this issue once it is available.

**Comment by mattmilburn** [comment]
Hi @jpouvrard @bora89 @bastianrickmannACN This is now fixed in the [latest v1.6.0 release](https://github.com/mattmilburn/strapi-plugin-menus/releases/tag/v1.6.0) of the plugin 🎉
